Update Query Insights grouping example response and response fields (#11890)
opensearch-project/documentation-website: _observing-your-data/query-insights/grouping-top-n-queries.md

`top_queries` | Array            | The list of top query groups.
`top_queries.timestamp` | Integer          | The execution timestamp for the first query in the query group.
`top_queries.id` | String           | The unique identifier for the query or query group.
`top_queries.total_shards` | Integer          | The number of shards on which the first query was executed.

> `top_queries.wlm_group_id` | String           | The Workload Management group ID for the first query in the query group.

Reviewer: ```suggestion
`top_queries.wlm_group_id` | String           | The workload management group ID for the first query in the query group.
```

`top_queries.query_group_hashcode` | String           | The hash code that uniquely identifies the query group and is generated from the [query structure](#grouping-queries-by-similarity).
`top_queries.task_resource_usages` | Array of objects | The resource usage breakdown for the various tasks belonging to the first query in the query group.
`top_queries.username` | String           | The username associated with the first query in the query group.
`top_queries.user_roles` | Array            | The security roles associated with the user who sent the first query in the query group.
`top_queries.node_id` | String           | The node ID of the node that coordinated the execution of the first query in the query group.
`top_queries.labels` | Object           | Used to label the top query.

> `top_queries.search_type` | String           | The search request execution type (`query_then_fetch` or `dfs_query_then_fetch`). For more information, see the `search_type` parameter in the [Search API documentation]({{site.url}}{{site.baseurl}}/api-reference/search/#query-parameters).

Reviewer: ```suggestion
`top_queries.search_type` | String           | The search request execution type. Valid values are `query_then_fetch` and `dfs_query_then_fetch`. See the `search_type` parameter in the [Search API documentation]({{site.url}}{{site.baseurl}}/api-reference/search/#query-parameters).
```
